setup.py

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I thought this file had to stay?

Reply:
I'll stick it back in (first time using the interactive git add functoinality on kraken)